service/src/main/java/uk/gov/hmcts/reform/fpl/model/Recipients.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class Recipients {
```

Comment:
We probably made a mistake by allowing the screens to capture the data in different way :(.